docs/userguide/storagedriver/device-mapper-driver.md

```diff
@@ -304,6 +304,35 @@ options were created in the previous steps.
  The last two lines also confirm that the correct devices are being used for 
 the `Data file` and the `Metadata file`.
 
+### Extend devicemapper thin-pool
+
+#### compile device_tool
+
+You have to compile contrib/docker-device-tool/device_tool.go [reference](../../../contrib/docker-device-tool/README.md)
```

Comment:
You need to change this to an absolute URL to https://github.com/....., because the "contrib" directory is not part of the documentation